server/ssrComponents/imageModal.html

```diff
@@ -0,0 +1,8 @@
+<div id="image-modal" class="image-modal">
```

Comment:
IMO this seems fine to just hardcode in the HTML for the article page. IMO it's not a heafty enough component that I think it warrants needing to be selectively added in the HTML formatter.

Reply:
👍